modules/apps/site/site-cms-site-initializer/src/main/java/com/liferay/site/cms/site/initializer/internal/display/context/BaseSectionDisplayContext.java

```diff
@@ -103,7 +103,7 @@ public String getAPIURL() {
 			sb.append(cmsSectionFilterString);
 		}
 
-		sb.append("))&nestedFields=embedded");
+		sb.append("&nestedFields=embedded,file.thumbnailURL");
```

Comment:
Why have you removed these two parenthesis? Now the OData expression is not syntactically correct (it has unclosed nested expressions).

Reply:
I've just saw it. Changed and pushed again.